demo-app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
-        android:usesCleartextTraffic="true" >
```

Comment:
Hi, it's a suggested setup by google as [here](https://developer.android.com/training/articles/security-config). The secure leak without it is also mentioned [here](https://medium.com/@son.rommer/fix-cleartext-traffic-error-in-android-9-pie-2f4e9e2235e6). However the exact reason, I'm not sure. Perhaps the best way is to change all traffic towards via https.

Reply:
Yeah I know about adding clearText but what am saying is it's already there as a manifest attribute so we don't need to extract it out to network config. Yeah the best way would be to use https but since for demo purposes we ran it on localhost which isn't on secure connection

Reply:
I think the point is to specifically define the whitelist url instead leaking clearText port to any urls. But ok, if this is solely for localhost running, then may be not necessary.

Reply:
Yeah morw so because the url is not predetermined